runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenConfig.java

```diff
@@ -22,19 +22,39 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package net.runelite.client.plugins.usernamesyncer;
+package net.runelite.client.plugins.loginscreen;
 
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup(
-	keyName = "rememberusername",
-	name = "Username Syncer",
-	description = "Syncs the username that is currently remembered between computers"
+	keyName = "loginscreen",
```

Comment:
I know the project's not consistent on this, but config keys should be camel-cased for readability.

Reply:
I actually think camelCasing the properties is anti-pattern, that we just adapted to use for w/e reason